### PR TITLE
Fix: Show background behind nav buttons [ED-16427]

### PR DESCRIPTION
**Pull request by max-zu**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

## Description
removed background on whole nav button container and show it behind the nav button itself across all browsers

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 1]
```diff
@@ -242,9 +242,19 @@
 		}
 
 		@media (max-width: $screen-tablet-max) {
-			&:active {
-				-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
-				-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+			&-prev, &-next {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:245]:
```suggestion
			&-prev,
			&-next {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 2]
```diff
@@ -242,9 +242,19 @@
...
+				&:active {
+					-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
+					-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+				}
+
+				& svg, i {
+					cursor: pointer;
+					&:active {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:253]:
```suggestion

					&:active {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss (hunk 1 above)
Comment (hein-obox) [reply assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:245]: Usually we like to break it into multiple lines. I am seeing that hasn't been done consistently, but I would be nice if we stick to convention.

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 3]
```diff
@@ -242,9 +242,19 @@
...
-			&:active {
-				-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
-				-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+			&-prev, &-next {
+				&:active {
+					-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
+					-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+				}
+
+				& svg, i {
```
Comment (louiswol94) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:251]:
```suggestion
				& svg,
				i {
```

Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:251]:
```suggestion
				& svg,
				i {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss (hunk 2 above)
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:253]:
```suggestion

					&:active {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 4]
```diff
@@ -253,17 +263,25 @@
 				outline: none;
 			}
 
-			&:active {
-				-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
-				-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
-			}
-
 			&-prev, &-next {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:266]:
```suggestion
			&-prev,
			&-next {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 5]
```diff
@@ -253,17 +263,25 @@
...
+				&:active {
+					-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
+					-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+				}
+
 				width: 20%;
 				i {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:273]:
```suggestion

				i {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 6]
```diff
@@ -253,17 +263,25 @@
...
 					padding: 10px;
 					background-color: rgba(0, 0, 0, 0.5);
 				}
+
+				& svg, i {
+					cursor: pointer;
+					&:active {
```
Comment (louiswol94) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:280]:
```suggestion
					cursor: pointer;

					&:active {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 7]
```diff
@@ -253,17 +263,25 @@
...
+					-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+				}
+
 				width: 20%;
 				i {
 					padding: 10px;
 					background-color: rgba(0, 0, 0, 0.5);
 				}
+
+				& svg, i {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:278]:
```suggestion
				& svg,
				i {
```

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss (hunk 6 above)
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:280]:
```suggestion

					&:active {
```

**Review by hein-obox — APPROVED** [review]

**Pull request merged**